commands/admin.js

```diff
@@ -4,15 +4,19 @@ const { getGuild, saveGuild } = require('../helpers/guildData')
 const { foemp } = require('../helpers/foemp')
 const { BUTTON_STYLES, DEFAULT_TIMEOUT } = require('../helpers/constants')
 
-async function verifyAdmin (interaction) {
+async function verifyAdminAsync (interaction) {
   const user = interaction.member
   const guild = await getGuild(interaction.guildId)
 
   const isGuildAdmin = user.permissions.has(Permissions.FLAGS.ADMINISTRATOR)
   const isBotAdmin = guild.admins.includes(user.id)
 
   if (!isGuildAdmin && !isBotAdmin) {
-    await interaction.reply('https://gph.is/g/4w8PDNj')
+    if (interaction.isdeferred || interaction.replied) {
```

Comment:
There is an issue with this line of code. 
This is the error it causes:
```
error:   2022-04-29T14:35:56.225Z | Error: Error [INTERACTION_ALREADY_REPLIED]: The reply to this interaction has already been sent or deferred.
    at Object.error (/home/mintage/dev/nerdlandbot_js/helpers/logger.js:43:7)
    at Client.<anonymous> (/home/mintage/dev/nerdlandbot_js/nerdlandbot.js:98:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
/home/mintage/dev/nerdlandbot_js/node_modules/discord.js/src/structures/interfaces/InteractionResponses.js:90
    if (this.deferred || this.replied) throw new Error('INTERACTION_ALREADY_REPLIED');
                                             ^

Error [INTERACTION_ALREADY_REPLIED]: The reply to this interaction has already been sent or deferred.
    at CommandInteraction.reply (/home/mintage/dev/nerdlandbot_js/node_modules/discord.js/src/structures/interfaces/InteractionResponses.js:90:46)
    at Client.<anonymous> (/home/mintage/dev/nerdlandbot_js/nerdlandbot.js:100:23)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  [Symbol(code)]: 'INTERACTION_ALREADY_REPLIED'
}
```
The fix is simple:
```if (interaction.deferred || interaction.replied) {```